source/configure/site-configuration-settings.rst

```diff
@@ -426,14 +426,14 @@ Access the following configuration settings in the System Console by going to **
   :systemconsole: Site Configuration > Users and Teams
   :configjson: .TeamSettings.MaxUsersPerTeam
   :environment: MM_TEAMSETTINGS_MAXUSERSPERTEAM
-  :description: The maximum total number of users per team, including active and inactive users. Default is **50** users per team.
+  :description: The maximum total number of users per team, including activated and inactive users. Default is **50** users per team.
```

Comment:
activated and deactivated users

We'll also need to update mentions of "inactive users" when it actually means "deactivated users"

Reply:
Thanks!